Capture/Capture Script/traffic_capture.php

```diff
@@ -64,8 +60,29 @@
 if(empty($headers))
 	$headers = "INVALID:HEADERS";
 
-//if the time of the current request falls outside of the timeframe, exit the script without logging.
```

Comment:
I assume this is being handled elsewhere since this was written? Can't remember where exactly